### PR TITLE
Update OVA to use photon 2.0

### PR DESCRIPTION
**Pull request by kars7e**
* move ova infra to top-level directory from scripts subdirectory
* Use dispatch image for building OVA based on photon 2.0
* Update scripts to work properly on photon 2.0
* Fix ovfenv binary to work properly in fusion
* Fix minor bugs causing errors in OVA logs

photon 2.0-based OVA is ~30% smaller than the previous one

Tested on Fusion 11.0 and VMC

### CODE REVIEW

File: ova/bootable/build-app.sh [hunk 1]
```diff
@@ -34,17 +34,23 @@ chage -I -1 -m 0 -M 99999 -E -1 root
 
 log3 "configuring ${brprpl}UTC${reset} timezone"
 ln --force --symbolic /usr/share/zoneinfo/UTC /etc/localtime
-log3 "configuring ${brprpl}en_US.UTF-8${reset} locale"
-/usr/bin/touch /etc/locale.conf
-/bin/echo "LANG=en_US.UTF-8" > /etc/locale.conf
-/sbin/locale-gen.sh
+
+## TODO: This causes issues on photon 2.0, debug or remove permanently when root cause
```
Comment (zjs) [inline ova/bootable/build-app.sh]: Should we file as issue to track this? (I'm unsure of the Dispatch conventions here, but that may make it easier to associate this with VIC; they'll be looking at Photon 2.0 for 1.5.)

**Comment by kars7e** [comment]
Thanks Zach!

File: ova/bootable/build-app.sh (hunk 1 above)
Comment (kars7e) [reply ova/bootable/build-app.sh]: Sounds good, thanks for reminding me!

Comment (DanielXiao) [reply ova/bootable/build-app.sh]: You can try to install glibc-i18n package.

**Pull request merged**